src/amo/components/AddonReviewCard/index.js

```diff
@@ -317,6 +318,14 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
           </span>
         );
       }
+      byLine = (
+        <Link
```

Comment:
The issue calls for only linking [the timestamp](https://github.com/mozilla/addons-frontend/blob/442ee43fafba36f4c27fb693f23bd417c87b4011/src/amo/components/AddonReviewCard/index.js#L304) which I like better. What do you think? Using a timestamp for permalinks is common practice.

Reply:
Yeah, I was being lazy. I agree that's better, but it means a more complicated l10n string, because we now need to embed the link in the middle of the string. Either that, or make it 2 different strings, but that would go against i18n philosophy, I think.

I'll try going the first route, embedding the link in the string.